modules/custom/reserve_movies/js/ShowMoviesByGenre.js

```diff
@@ -17,8 +17,23 @@ function ShowMoviesByGenre() {
     });
 
     $(".movie-class").on("click", function(){
+        const popUpID = "#popup-" + $(this).attr("id");
+        const reservationButtonID = "#reservation-button-" + $(this).attr("id");
+        const confirmReservationButton = "#confirm-reservation-button-" + $(this).attr("id");
+        const cancelPopupSelection = "#cancel-" + $(this).attr("id");
```

Comment:
Sta je ovde isto?

Reply:
Mogu da izvucem $(this).attr("id") u neku promenljivu da se ne ponavljam